Lil-Peep/src/styles/modules/site-collection.scss

```diff
@@ -219,26 +219,18 @@
     background-color: #{$view-green};
     border-radius: 3px;
     border-style: none;
-    padding: 10px 0;
+    color: #{$color-body};
+    display: block;
+    font-family: 'Oswald', sans-serif;
```

Comment:
I think that can be better with a variable.scss.

'Oswald',sans-serif -> $oswald

Reply:
Got it